include/fastdds/statistics/monitorservice_types.idl

```diff
@@ -0,0 +1,131 @@
+// Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file include/fastdds/statistics/ms_types.idl
```

Comment:
I think it is enough to put only `@file`. Nevertheless, this is related to Doxygen documentation and I think it is not applicable to IDL files. If kept, then please update the file name to `monitorservice_types.idl`